From 80c99317f1bfb4333757cad0819924f83f62b24e Mon Sep 17 00:00:00 2001
From: Barry Warsaw <barry@canonical.com>
Date: Sat, 15 Feb 2014 22:47:01 +0000
Subject: python: Disable import parsing by default

Parsing imports mixes up imports with actual class definitions.  If you need
to re-enable this, use the --python-kinds=+i option.

Bug: http://sourceforge.net/tracker/?func=detail&aid=2829739&group_id=6556&atid=106556
Bug-Ubuntu: https://bugs.launchpad.net/bugs/618979
Reviewed-by: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no
Last-Update: 2010-08-17

Patch-Name: python-disable-imports.patch
---
 python.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python.c b/python.c
index a90d072..bf797de 100644
--- a/python.c
+++ b/python.c
@@ -56,7 +56,7 @@ static kindOption PythonKinds[] = {
 	{TRUE, 'f', "function", "functions"},
 	{TRUE, 'm', "member",   "class members"},
     {TRUE, 'v', "variable", "variables"},
-    {TRUE, 'i', "namespace", "imports"}
+    {FALSE, 'i', "namespace", "imports"}
 };
 
 static char const * const singletriple = "'''";
